### ISSUE TITLE
Does not support Python 3.8

### ISSUE THREAD

**Issue by nkrot** [savioxavier/pyboxen#2]
Hallo,

Dispite the claim made in `pyproject,.toml` to support Python 3.8, this "Incredibly customizable terminal boxes for Python" does not work with Python 3.8. Obviously.

The problem begins when importing the library:

```
    ALL_BOXES: dict[str, Box] = {
TypeError: 'type' object is not subscriptable

```

Reason: the syntax `dict[str, Box]`, intead of `Dict[str, Box]` first appeared in Python 3.9. Please check here: https://www.python.org/downloads/release/python-390/ (mention of https://peps.python.org/pep-0585/)

To make this library usable with Python 3.8, it is sufficient to replace some type annotations

```
dict[...] --> Dict[...]
tuple[...] --> Tuple[...]
```

and extend import statement correspondingly

```
from typing import Dict, Tuple
```

**Comment by savioxavier** [comment]
Thanks for reporting this issue.
I had to bump the version down from 3.10 to 3.8 as a result of someone stating that they wanted it to work with an earlier version. Since it was just a matter of a single line diff, I did not feel the need to test it out. Moreover, the creator of that original issue said that everything worked as fine, and I left it at that.
I now suspect, however, that the person was using a 3.9 version of Python where this issue was not present. I'll make the diff as soon as possible and release a minor version.

**Comment by savioxavier** [comment]
I think I made a fix for this in the `typing-fix` branch right now. Before I publish a release, is it possible to "pull" the branch and verify whether the problem is still reproducible?

branch: [`https://github.com/savioxavier/pyboxen/tree/typing-fix`](https://github.com/savioxavier/pyboxen/tree/typing-fix)

**Comment by nkrot** [comment]
Hi
Thanks for addressing this issue promptly. I have tested the branch `typing-fix` with python 3.8.10 and confirm the fix works.

![pyboxen typing-fix for python 3 8 2024-02-05](https://github.com/savioxavier/pyboxen/assets/1128218/857160b6-a6be-416f-b146-7a72a3ebafff)

**Comment by nkrot** [comment]
Hi!
Thanks for addressing this issue promptly. I have tested the branch `typing-fix` and can confirm it works.

![pyboxen typing-fix for python 3 8](https://github.com/savioxavier/pyboxen/assets/1128218/e04f3ce8-8ce3-409e-a0a8-4e853e9fcc5f)

Please remember to update the version number :)

**Comment by savioxavier** [comment]
Published a new release [`v1.3.0`](https://github.com/savioxavier/pyboxen/releases/tag/v1.3.0). I've made it a "minor" version bump since it technically has some ~~reverse~~ breaking changes.

Once again, thank you so much for reporting this issue!
